pkg/api/v1alpha1/tinkerbelldatacenterconfig.go

```diff
@@ -22,7 +22,9 @@ func NewTinkerbellDatacenterConfigGenerate(clusterName string) *TinkerbellDatace
 		ObjectMeta: ObjectMeta{
 			Name: clusterName,
 		},
-		Spec: TinkerbellDatacenterConfigSpec{},
+		Spec: TinkerbellDatacenterConfigSpec{
+			TinkerbellIP: "",
```

Comment:
isn't `TinkerbellIP` optional for workload clusters?

Reply:
Doesn't look like it's an optional [field](https://github.com/aws/eks-anywhere/blame/5acb0dacb527185383444e9f85ec3194a5818004/pkg/api/v1alpha1/tinkerbelldatacenterconfig_types.go#L13) .? I think just that it has to be same as Management cluster for workload clusters .?